### ISSUE TITLE
[Accessibility Audit] Code Intel: Inspecting auto-indexing jobs for a repository

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#33513]
### Steps to replicate journey

Taken from the [user journey list document](https://docs.google.com/document/d/1kA6aVOAgID_uPm-d6uEC1DG6WHjJ8fPT0UAXQcML_KQ/edit?usp=sharing). Use this for further context.

- Navigate to a repository in Sourcegraph.
- Click on the 🧠 Code Intelligence button.
- Click on Auto-indexing.
- Potential options:
  - Run:
    - [Optional: Adjust Index state filter]
    - Click Enqueue
  - Inspect:
    - [Optional: Change Index state filter.]
    - [Optional: Search indexes for specific ones.]
    - Click on > for an index.
    - Potential options:
      - Click on 🗑 Delete index button.
      - Inspect steps:
        - Expand step.
        - Expand Log output and potentially scroll to read it.
        - [Optional: Collapse Log output.]
        - Collapse step.
      - Click on > next to Upload.

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

**Note:** We are planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

### Any other questions?

See the [audit FAQ](https://docs.google.com/document/d/1nCi7TvYRpj9eG0_QtgqK1c4L28EiJKVuxyuvg1lAGgo/edit#) - add your question if it isn't there already!

**Comment by sourcegraph-bot-2** [comment]
Heads up @macraig - the "team/code-intelligence" label was applied to this issue.

**Comment by efritz** [comment]
Global:

- Every text element announces itself as clickable. Is this correct?

When trying to navigate to the code intel page:

Skipping down headings went from sg -> repo name -> "files and directories" -> "changes" confused us (when audio only) about the hierarchy of the page. Resorted to tab navigation to get to the settings page. Suggestion: make the repository navigation bar a more significant part of the page hierarchy (not sure since we're redesigning this page).

Incidental issues on the repo page:

- [ ] "Istio Automation" svg for authorship in the changes section is pronounced as "eeh - aah". Like WTF. Put an svg area label "author avatar" or something instead.
- [ ] The github svg before the repo name also does not have a meaning and announces itself as a generic image.

On the code intel uploads page:

- [ ] Once on the auto-indexing jobs page, the "enqueue" bar is jarring. It doesn't have any instructions and announces itself as "Git revlike". Similarly the text input doesn't say what it's doing. Add some text and headings around this area.
- [ ] When already on the page the side navbar does not announce or refocus.
- [ ] Enqueue is pronounced as "uhn kwwoo" like a moron
- [ ] "Index state" announces the filter but not that it's part of a list.
- [ ] The dropdown is incorrectly labeled as "sort-notebooks"
- [ ] The metadata about an index "Directory / indexed at commit asldfkjsafd by sldkj" is not readable as it segments all of the links from text elements. Is there a way to read the entire thing instead?
- [ ] Link image announces the graphql ID and should instead say instead the navigation target in human understandable terms
- [ ] Each of the groups laying out auto indexing records aren't self-explanatory. A table would better way to lay out the metadata at the top.
- [ ] Timeline images don't have any meaning
- [ ] The timeline view has "5 minutes" between two states, which isn't explained (it's a bit visually confusing as well).
- [ ] Collapsible sections here
- [ ] After expanding a setup step, it started blathering "git -C /tmp/10981203 init" which wasn't announced as a script part of the view. Be sure to name each of the code blocks "we have log output" which does well to ground the user.
- [ ] No headings for timeline settings. No fast way to jump from the middle of git pull logs.
- [ ] "This job uploaded an index" should be renamed
- [ ] Step numbers should be announced as part of the name, and the time should be separated with a verb "started 1 hour ago". This will get around the wonky phrasing "index 1 hour ago".
- [ ] Minimize the data about the associated upload, as it's a bit confusing to read on the context of this page
- [ ] First thing announced on the upload/index pages are "Upload processed successfully" or flash images. I think it's because these have heading elements that change the hierarchy in a significant way.
- [ ] The flash message for "upload/index for commit XXX" are not announced because the "repository commit graph" status alert takes precedence for some weird reason.
